### ISSUE TITLE
Version 4.5.0 Crashes website

### ISSUE THREAD

**Issue by domi-87** [cloudflare/Cloudflare-WordPress#419]
### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the plugin and it is still present.

### WordPress version

5.7.2

### Cloudflare-WordPress version

4.5.0

### PHP version

8.0.5

### Expected result

No Errors

### Actual result

An error of type E_ERROR was caused in line 15 of the file /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/symfony/polyfill-intl-normalizer/bootstrap.php. Error message: Uncaught Error: Failed opening required ‘/var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/symfony/polyfill-intl-normalizer/bootstrap80.php’ (include_path=’.:/usr/share/php’) in /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/symfony/polyfill-intl-normalizer/bootstrap.php:15
Stack trace:
#0 /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/composer/autoload_real.php(69): require()
#1 /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/composer/autoload_real.php(59): composerRequire386a6fce91e17b37d63cb251597d5bd5()
#2 /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/vendor/autoload.php(7): ComposerAutoloaderInit386a6fce91e17b37d63cb251597d5bd5::getLoader()
#3 /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/cloudflare.loader.php(3): require_once(‘…’)
#4 /var/www/[redacted]/public_html/wp-content/plugins/cloudflare/cloudflare.php(49): require_once(‘…’)
#5 /var/www/[redacted]/public_html/wp-settings.php(391): include_once(‘…’)
#6 /var/www/[redacted]/public_html/wp-config.php(126): require_once(‘…’)
#7 /var/www/[redacted]/public_html/wp-load.php(37): require_once(‘…’)
#8 /var/www/[redacted]/public_html/wp-admin/admin-ajax.php(22): require_once(‘…’)
#9 {main}
thrown

### Steps to reproduce

1. Activate the Plugin

**Comment by gboudreau** [comment]
Duplicates #415
